src/lib/endpoint.ts

```diff
@@ -92,6 +122,18 @@ export class Endpoint {
     return ([] as PacketWithMetadata[]).concat(...resultsNested);
   }
 
+  // returns all packets (auto-paginates, so be careful about not setting a minHeight)
+  public async querySentPackets({
+    minHeight,
+    maxHeight,
+  }: QueryOpts = {}): Promise<PacketWithMetadata[]> {
+    const txsPackets = await this.getPacketsFromTxs({ minHeight, maxHeight });
+    const eventsPackets = await this.getPacketsFromBlockEvents({ minHeight, maxHeight });
```

Comment:
This will throw an unspecified (and unhandled) error if used on any chain less than tendermint v0.34.9 (cosmos-sdk v0.42.4). I would tend to making this more robust, unless there really is no need to handle earlier chains.

See comments on https://github.com/cosmos/cosmjs/pull/810#pullrequestreview-667718860